src/App.vue

```diff
@@ -23,10 +23,33 @@
       </shutdown>
       <loading v-else-if="loading" :progress="loadingProgress">
         <small class="text-muted w-75 text-center">{{ loadingText }}</small>
+        <div v-if="showDebugOption">
+          <small class="text-muted w-75 text-center">If the loading is stuck, click here to debug : </small>
```

Comment:
```suggestion
          <small class="text-muted w-75 text-center">If the loading is stuck, click here to debug: </small>
```

There shouldn't be a space here: http://www.sussex.ac.uk/informatics/punctuation/colonandsemi/colon

Reply:
Sorry, french typo haha